setup_wordpress.sh

```diff
@@ -1,3 +1,90 @@
 #!/bin/bash
+set -e
+echo '################## LEMP Stack Installation ####################'
+echo "------------------ Updating APT ------------------"
+sudo apt-get -y update && sudo apt-get -y upgrade
 
-echo 'This script should install and setup Wordpress'
+echo "------------------ Installing Nginx ------------------"
+sudo apt-get -y install nginx
+
+echo "------------------ Configuring Firewall ------------------"
+sudo ufw allow 'Nginx Full'
```

Comment:
Can you tell how does this program work under the hood?

Reply:
After installing Nginx, the necessary ports need to be opened to handle the traffic. Because I also wanted to use the HTTPS, I add the app 'Nginx Full' to the firewall which opens both port 80 (Nginx HTTP) and port 443 (Nginx HTTPS). with the redirection, whenever there's a request for HTTP, it will be redirected to the HTTPS which is a safer protocol. (Configured with SSL)